swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -243,15 +247,14 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
       sendChunk(tunnelLogArchive)
 
     case .idle:
-      guard let logFolderURL = SharedAccess.logFolderURL,
-        let logFolderPath = FilePath(logFolderURL)
+      guard let logFolderURL = SharedAccess.connlibLogFolderURL
```

Comment:
This wasn't pointing to the `connlib` directory but to the parent `logs` directory.

Reply:
This should be reverted because it will skip any Swift logs we make in the Network Extension

Reply:
Oh, didn't realise that. If we change the path back you need to adjust the path to the sym link.